Daytime-Operations/Auxiliary-Telescope/AT-Calibrations/LATISS-Combined-Calibrations-Procedure/latiss-combined-calibrations-procedure.rst

```diff
@@ -13,30 +13,33 @@ Latiss Combined Calibrations Generation Procedure
+
+For at least one type of test (as defined in `DMTN-101`_), if the majority of tests fail verification in the majority of detectors and the majority of exposures, then the script will terminate after calculating these verification statistics. If ``generate_calibrations`` is ``True``, the calibration will not be certified into the ``calibration_collection`` collection. The configuration parameters ``number_verification_tests_threshold_bias``, ``number_verification_tests_threshold_dark``, and ``number_verification_tests_threshold_flat`` will be used to define thresholds to decide whether the calibration will pass verification and should be certified or not. Currently, verification is only implemented for ``BIAS``, ``DARK``, and ``FLAT`` calibration types. If the configuration parameters ``do_defects`` and ``do_ptc`` are set to ``True``, verification will be skipped for the ``DEFECTS`` and ``PTC`` calibrations and they will be automatically certified.
 
 The script currently has the option (via the ``script_mode`` parameter in the configuration options) to:
 
 - take only biases, 
 - take biases and darks, and 
 - take biases, darks, and flats. 
   
-These options are constrained by the fact that the generation or construction of one calibration depends on the existence of the previous one (i.e., to generate a combined dark, a combined bias is necessary, and to generate a flat, a combined dark and a combined bias are necessary). Calibration generation from the images taken can be skipped by setting ``generate_calibrations``. This will speed up the execution time of the script, and subsequent tasks (for example, verification tasks or the PTC construction task) will look for necessary calibrations in their input collections (whose default is the standard calibrations collection: ``LATISS/calib``).
+These options are constrained by the fact that the generation or construction of one calibration depends on the existence of the previous one (i.e., to generate a combined dark, a combined bias is necessary, and to generate a flat, a combined dark and a combined bias are necessary). If ``generate_calibrations`` is ``True``, calibrations will eventually end up in the collection specified by the ``calib_collection`` configuration parameter and, if certified, will have a validity span range from ``certify_calib_begin_date`` to ``certify_calib_end_date``.
```

Comment:
If ``generate_calibrations`` is ``True``, and the external and internal verifications are successful, then the ......

Reply:
(not what we want to do regularly, but it's an option in the code that has been useful for testing).